### PR TITLE
Fix docs as says bug 9085

### LINKED ISSUE #9085: [8.0][report] PDF reports get no styles
All the [Useful Remarks](https://www.odoo.com/documentation/8.0/reference/reports.html#useful-remarks) section from the docs is lying:

> Twitter Bootstrap and FontAwesome classes can be used in your report template

It only happens in HTML reports.

This is a demo sale order, printed normally from Odoo (PDF, then screen capture):
captura de pantalla de 2015-10-15 13-53-07

This is the same sale order, but printed by visiting `http://localhost/report/html/sale.report_saleorder/6` and then printing to PDF from Firefox:
captura de pantalla de 2015-10-15 13-53-32

The difference is quite obvious. PDF reports get no styles at all.

> Local CSS can be put directly in the template

If this refers to the `style` attribute, it's OK. But one cannot put directly `<style>` elements in the template. The phrasing is misleading.

> Global CSS can be inserted in the main report layout by inheriting its template and inserting your CSS:
> [code]

This is also not working, but I imagine it's because of the first point.

It seems fixed in 9.0 though.

### PR DESCRIPTION
**Pull request by yajo**
This fixes #9085 (well... documents it).

### CODE REVIEW

**Comment by Yajo** [comment]
cc @sle-odoo

**Comment by sle-odoo** [comment]
i think you don't get it entirely. It's not a "false doc" (as your commit message tells) nor it is a "lie" (as you wrote in #9085). You deleted a sample code explaining how to add css rules available in the reports by documenting how to troubleshoot a common issue in an odoo setup. While i agree the information you add are not superfluous, you have to keep the first part (and maybe put it in a danger block && finish it with how to add a link to a stylesheet).

Also, i'm not okay with the wording ("awesomely ugly", why not simply "if it appears that your report is missing the styles (i.e. the text appears but the style/layout is different from the html version)"). You could also write that, to know if it is actually this issue, it is useful to check the server logs and see that the server does not receive request for the report's assets.

There is also a typo, the last "web.base.url" should be "report.url". And maybe you could document that web.base.url is reset everytime the admin is logged (to check) and whate does the web.base.url.freeze key do too? So it i's complete.

Regards

**Comment by sle-odoo** [comment]
i mean put the troubleshoot part in a danger block. and now that i think about it, it may not be the right place for this text (troubleshoot part).. i guess it should land in the deploy guide, next to the version of wkhtmltopdf supported.

**Comment by Yajo** [comment]
I agree to all, except that I deleted instructions that, if followed, did not work. I have not found any alternative way to do it, so I deleted it to avoid confusions.

I'll fix all other things, but if you want me to restore that paragraph, please provide a workaround or fix.

File: doc/howtos/backend.rst [hunk 1]
```diff
@@ -1622,6 +1622,28 @@ http://localhost:8069/report/html/account.report_invoice/1 (if ``account`` is
 installed) and the PDF version through
 http://localhost:8069/report/pdf/account.report_invoice/1.
 
+.. _reference/backend/reporting/printed-reports/pdf-without-styles:
```
Comment (Yajo) [inline doc/howtos/backend.rst]: All done now. Is this OK? I never wrote such things with Sphinx...

**Comment by sle-odoo** [comment]
yes you are right, actually i we have to make an xpath expr="." position="inside" inherit_id="report.style" to directly add css rules, but this doesn't work with lxml/our xpath things so we have to put the css rules in a t tag

```
<t>
.rule1{color:red;}
</t>
```

this should work. Could you try it please?

**Comment by Yajo** [comment]
That works, let me update this.

**Comment by rafaelbn** [comment]
Hi, how is the state of this PR? It looks :+1:

**Comment by Yajo** [comment]
I don't want to bother anyone, but this is ready to merge IMHO.

**Comment by sle-odoo** [comment]
merged at 4eab4d786805ecfd9bdff0b013ff831c4984cc99

thanks a lot!

**Pull request closed without merging**